cypress/support/commands.js

```diff
@@ -23,3 +24,31 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', () => {
```

Comment:
When using the `cy.login()` function in the test file, you're allowing the user of this command to pass the user and password, but here you're not accepting those arguments.
I suggest that you accept them (maybe a user object), but default the values to the info coming from the `cypress.env.json` file. Something like this:

```js
Cypress.Commands.add('login', (user = {
  email: Cypress.env("user"),
  password: Cypress.env("password")
}) => {
  cy.visit('/login')
  cy.get('#email')
    .should('be.visible')
    .type(user.email)
  cy.get('#password')
    .should('be.visible')
    .type(user.password)
  cy.get('button[type="submit"]')
    .should('be.visible')
    .click()
})
```

Reply:
Done!